COLLABORATOR_GUIDE.md

```diff
@@ -64,7 +64,8 @@ from other Collaborators. Leave at least 48 hours during the week and
 72 hours over weekends to account for international time differences
 and work schedules. Trivial changes (e.g. those which fix minor bugs
 or improve performance without affecting API or causing other
-wide-reaching impact) may be landed after a shorter delay.
+wide-reaching impact), and focused changes that affect only documentation
+and/or the test suite, may be landed after a shorter delay.
```

Comment:
I would add "if the change is approved by at least 2 collaborators". This is the typical case anyway.

Reply:
@mcollina done